src/devices/Pca95x4/Pca95x4.cs

```diff
@@ -0,0 +1,199 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Gpio;
+using System.Device.I2c;
+
+namespace Iot.Device.Pca95x4
+{
+    /// <summary>
+    /// A general purpose parallel I/O expansion for I2C applications.
+    /// </summary>
+    public class Pca95x4 : IDisposable
+    {
+        private readonly I2cDevice _i2cDevice;
+        private GpioController _masterGpioController;
+        private readonly int? _interrupt;
+
+        /// <summary>
+        /// Initializes new instance of Pca95x4.
+        /// A general purpose parallel I/O expansion for I2C applications.
+        /// </summary>
+        /// <param name="i2cDevice">The I2C device used for communication.</param>
+        /// <param name="interrupt">The input pin number that is connected to the interrupt (INT).</param>
```

Comment:
I'm not too familiar with the device so this might be a dumb question, but when would you want to add an interrupt? Is it to detect changes on values?

Reply:
It's similar to the MCP23xxx where you can setup interrupts where if any of the inputs change, it will send the event via the Interrupt pin so it can notify the master device to do work.  Once you read/clear what is needed by binding, the Interrupt pin goes back to normal state until next interrupt condition occurs on inputs that are configured.

I have not studied/tried the GPIO interrupt scheme yet.  Planned to do soon and then add the event code to the Mcp23xxx and this binding later.

So right now, you can only just read the Interrupt pin with the binding helper method.